src/pages/index.js

```diff
@@ -33,3 +110,38 @@ const IndexPage = () => {
 }
 
 export default IndexPage
+
+export const homeQuery = graphql`
+query {
+  hill: file(relativePath: { eq: "other/hill.jpg"}){
+    childImageSharp{
+      fluid(quality: 70, maxWidth: 1400){
+        ...GatsbyImageSharpFluid
```

Comment:
Can we use WebP here or does that cause issues with the background image component?